lib/asciidoctor/standoc/inline.rb

```diff
@@ -86,18 +86,18 @@ def inline_break(node)
       end
 
       def page_break(_node)
-        noko { |xml| xml.pagebreak }.join("\n")
+        noko(&:pagebreak).join("\n")
       end
 
       def thematic_break(_node)
-        noko { |xml| xml.hr }.join("\n")
+        noko(&:hr).join("\n")
```

Comment:
@opoudjis can you confirm this is correct?

Reply:
IDE did this for me I will revert it to keep the commit clean